integration/integration_test.go

```diff
@@ -0,0 +1,64 @@
+package integration
+
+import (
+	"testing"
+
+	"sigs.k8s.io/krew/integration/krew"
+)
+
+const (
+	// validPlugin is a valid plugin with a small download size
+	validPlugin = "konfig"
+)
+
+func TestInstall(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	krewTest, cleanup := krew.NewKrewTest(t)
+	defer cleanup()
+
+	krewTest.WithIndex().Cmd("install", validPlugin).RunOrFail()
+}
+
```

Comment:
i suspect at some point we’ll use a more advanced testing package that lets us classify tests by grouping them under types. I believe testify pkg helps with that. We can probably migrate later on.

Reply:
Agreed. We should make tests prettier when we have a better overview of what we need. For the first draft I don't think that's possible though.